docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,286 @@
+Ignores
+========
+
+As explained in section :ref:`gettingStartedLarva`, Larva-tests verify if values that are read from the tested Frank configuration are equal to the expected constant values. However, some tested systems produce different results each time that they are executed, even though the inputs are the same. Examples are systems that return a time stamp. In Larva, you can address this issue using ignores. With ignores, values returned from the tested system are transformed, before they are compared to the expected value. As a Frank developer, you have to choose these transformations wisely and include meaningful aspects of your returned value. 
+
```

Comment:
You write "... and include meaningful aspects of your returned value". I would choose the word "retain". When you say "include", you suggest that your test adds new information to the output of the system you are testing. That would not make sense, because you are testing the value you got. The word "retain" expresses this. You lose something by using ignores, but you keep the information you want to test.

The rest of this paragraph is very good.

Reply:
I checked that my proposal has been applied. Thanks.